.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://github.com/prisma/prisma2/discussions
```

```suggestion
    url: https://github.com/prisma/prisma2/discussions/new
```

Let's open a new discussion right away.